docs/admin/admission-controllers.md

```diff
@@ -101,14 +112,16 @@ If your cluster supports containers that run with escalated privileges, and you
 restrict the ability of end-users to exec commands in those containers, we strongly encourage
 enabling this plug-in.
 
-### GenericAdmissionWebhook (alpha)
+### GenericAdmissionWebhook (alpha in 1.8; beta in 1.9)
```

Comment:
We now have the following terms that appear to mean the same thing:
- extensible-admission-controllers
- external-admission-webhooks
- Dynamic Admission Control
- External Admission Webhooks

Reply:
I intend to fix the names before this is finalized.

Reply:
Chao is already on it: https://github.com/kubernetes/kubernetes/pull/55988

Reply:
It's called ValidatingAdmissionWebhook